devlib/utils/collections_abc.py

```diff
@@ -0,0 +1,25 @@
+#    Copyright 2018 ARM Limited
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+'''
+Give a common name to import collections.abc (Python 3) and collections
+(Python 2), since it si not supported by they "future" package.
+'''
+
+try:
+    from collections.abc import *
+except ImportError:
+    from collections import *
+
```

Comment:
The only benefit is sharing that logic with the next module that needs it, otherwise it could be "inlined". Catching `ImportError` is done on purpose since supporting `collections.abc` is on the "whishlist" of `future`, so any module like that providing Python 3-like support in Python 2 will be used if present. `collections.abc` is a subset of Python 2's `collections`, so it's better to let that be handled by a properly coded package like `future` if available.